appengine/endpoints-frameworks-v2/backend/README.md

```diff
@@ -28,11 +28,11 @@ To build the project:
 To generate the required configuration file `swagger.json`:
 
 0. Download and unzip the [Endpoints Framework tools
-package](http://search.maven.org/remotecontent?filepath=com/google/endpoints/endpoints-framework-tools/2.0.0-beta.4/endpoints-framework-tools-2.0.0-beta.4.zip).
+package](http://search.maven.org/remotecontent?filepath=com/google/endpoints/endpoints-framework-tools/2.0.0-beta.7/endpoints-framework-tools-2.0.0-beta.7.zip).
 
```

Comment:
Thanks for the suggestion. I think we will go with what we have for now. I will create a bug to explore this solution and update the sample.